content/docs/hardware/general/case/_index.de.md

```diff
@@ -170,9 +181,20 @@ liegen auf der [ADFC page - Vorlagen zum OpenBikeSensor ](https://aktiv.adfc.de/
 4. USB Cover:
 [OBS-MainCase-A-003_USB-Cover_v0.1.1.stl](https://github.com/openbikesensor/OpenBikeSensor3dPrintableCase/blob/master/MainCase/VerticalCase_JSN-AJ/OBS-MainCase-A-003_USB-Cover_v0.1.1.stl)
 5. Anschluß für die Fahrradhalterungen
-[OBS-Mounting-A-001_StandardOBSMount_v0.1.1.stl](https://github.com/openbikesensor/OpenBikeSensor3dPrintableCase/blob/master/MainCase/VerticalCase_JSN-AJ/OBS-Mounting-A-001_StandardOBSMount_v0.1.1.stl)
-
-{{< imgproc "UM3_curaUI_OBS-Maincase-all_pieces.jpg" Resize 800x >}}
+[OBS-Mounting-A-001_StandardOBSMount_v0.1.2.stl](https://github.com/openbikesensor/OpenBikeSensor3dPrintableCase/blob/master/MainCase/VerticalCase_JSN-AJ/OBS-Mounting-A-001_StandardOBSMount_v0.1.2.stl). 
+  {{< imgproc "Mounting-A-001.png" Resize 800x >}} 
+  Der Anschluss für die Fahradhalterung ist das Gehäuseteil mit der höchsten 
+  Belastung, es wird deshalb empfohlen, für dieses Teil mit höherem 
+  Infill (>50%) zu drucken, und die Wandstärke auf 2.4mm zu erhöhen, 
+  respektive je nach Slicer bei einer 0.4mm Düse 6 Perimeter zu wählen. Das
+  Teil sollte seitlich stehend gedruckt werden, um die Extrusionsrichtung
+  in Belastungsrichtung auszurichten, was das Teil zusätzlich verstärkt. Bei
+  Haftungsproblemen kann das Teil in den Slicereinstellungen mit einem Brim
+  versehen werden.
+  {{< /imgproc >}}
+
+
+{{< imgproc "case parts - prusaslicer.png" Resize 800x >}}
```

Comment:
This image seems to be missing.

Reply:
It's part of the pr (bottom most file in list of Files changed)

Reply:
User error on my side.